src/Utilities.h

```diff
@@ -28,6 +38,40 @@ double sse_calc(const std::vector<T>& simval, const std::vector<T>& obsval){
 	
 }
 
+/**
+ * @brief Create new vector summating every Nth element
+ * 
+ * Summate values at every Nth index, e.g. n = 7 would summate
+ * daily data to be weekly data
+ * 
+ * @param data_vector	Vector of data to be reduced
+ * @param n	Nth value at which summation should occur (i.e. index+1)
+ * 
+ * @return Vector of summation values
+ */
+template<typename T>
+std::vector<T> AccumulateEveryNth(const std::vector<T>& data_vector, const int& n)
```

Comment:
The name should really be `AccumulateEveryN` rather than `AcculumateEveryNth`. The latter sounds like it only extracts every nth element, when in fact it sums elements in blocks of n.